src/kata6.joinNames.js

```diff
@@ -1,3 +1,10 @@
-const joinNames = namesObj => {};
+const joinNames = namesObj => {
+  const stringOfNames = namesObj.reduce((accumulator, person, index) => {
```

Comment:
Great stuff, nice use of reduce and demonstration of understanding the parameters.